src/components/Datagrid/Datagrid.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import { GridCell, GridCellKind, Item } from "@glideapps/glide-data-grid";
+import { Button } from "@saleor/macaw-ui";
+import Decorator from "@saleor/storybook/Decorator";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import Datagrid from "./Datagrid";
+import { initialData } from "./fixtures";
+import useDatagridChange from "./useDatagridChange";
+
+const errors = [{ id: "9", field: "balance" }];
+
+const availableColumns = [
+  { title: "ID", id: "uuid", width: 300 },
+  { title: "Loaned", id: "loan-active", width: 70 },
+  { title: "Loaned Amount", id: "loan", width: 200 },
+  { title: "Name", id: "name", width: 200 },
+  { title: "Balance", id: "balance", width: 200 },
+  { title: "Eye color", id: "eyeColor", width: 200 },
+  { title: "Age", id: "age", width: 80 },
+] as const;
```

Comment:
I think this should be in `fixtures` file. Wdyt?

Reply:
This is something you'd be tweaking in a component itself and does not come from API, so I'd say this should stay here